internal/jennies/java/rawtypes.go

```diff
@@ -298,3 +299,30 @@ func (jenny RawTypes) getVariant(t ast.Type) string {
 	}
 	return variant
 }
+
+func (jenny RawTypes) defaultConstructor(object ast.Object) []ast.Argument {
```

Comment:
Why not generating constructors for every object?

Reply:
Because it generates monster builders that they aren't used in builders... So at the moment I didn't want to add them. I preferred to add them mainly to the builders that we omit in veneers because they have a few fields only.